services/blockchain-indexer/shared/dataService/blocks.js

```diff
@@ -84,12 +76,44 @@ const getBlocks = async (params = {}) => {
 		total = await getTotalNumberOfBlocks();
 	}
 
+	return total;
+};
+
+const formatBlock = async block => {
+	const blocksResponse = {
+		data: [],
+		meta: {},
+	};
+
+	const response = await business.formatBlock(block);
+	blocksResponse.data.push(response);
+
+	return {
+		data: blocksResponse.data,
+		meta: {
+			count: 1,
```

Comment:
Isent this blocksResponse.data.length?
```suggestion
			count: blocksResponse.data.length,
```